pkg/deployments/tasks/20220707-distribution-scheduler/readme.md

```diff
@@ -5,5 +5,6 @@ Deployment of the `DistributionScheduler`, which can be used to schedule token d
 ## Useful Files
 
 - [Ethereum mainnet addresses](./output/mainnet.json)
+- [Avalanche mainnet addresses](./output/avalanche.json)
```

Comment:
Hmm the entry here is correct, but the fact that it is not deployed in other L2s makes me think that this is probably not required in Avalanche, and it shouldn't be considered a 'canonical' deployment.

Reply:
Yes; I'm not sure why that was deployed there. (It wasn't one of the recent ones I just did.) Anyway not hurting anything.